### PR TITLE
[11.0][MIG] account_type_menu

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: account_type_menu/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+# -*- coding: utf-8 -*-
```
Comment (simahawk) [inline account_type_menu/__init__.py]: remove coding headers

File: account_type_menu/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Account Types Menu',
+    'version': '11.0.1.0.0',
+    'category': 'Accounting',
+    'license': 'AGPL-3',
+    'summary': "Adds a menu entry for Account Types",
+    'author': "Akretion,Odoo Community Association (OCA)",
+    'website': 'http://www.akretion.com/',
```
Comment (simahawk) [inline account_type_menu/__manifest__.py]: https://github.com/OCA/account-financial-tools

File: account_type_menu/README.rst [hunk 3]
```diff
@@ -0,0 +1,47 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==================
+Account Types Menu
+==================
+
+This module adds a menu entry *Account Type* under *Accounting > Configuration > Accounting*, because this menu entry doesn't exists in the official *account* module of Odoo 10.
```
Comment (simahawk) [inline account_type_menu/README.rst]: `of Odoo 10` -> `of Odoo`.

**Comment by simahawk** [comment]
dependency for https://github.com/OCA/account-analytic/pull/141

File: account_type_menu/__init__.py (hunk 1 above)
Comment (grindtildeath) [inline account_type_menu/__init__.py]: To remove

File: account_type_menu/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
```
Comment (grindtildeath) [inline account_type_menu/__manifest__.py]: To remove

**Review by grindtildeath — CHANGES_REQUESTED** [review]
Only two small changes, otherwise looks good ! :+1:

**Review by simahawk — CHANGES_REQUESTED** [review]

**Review by simahawk — APPROVED** [review]

**Comment by simahawk** [comment]
@grindtildeath ping :)

**Review by grindtildeath — APPROVED** [review]

**Pull request merged**